plugin/wintabs.vim

```diff
@@ -69,6 +69,9 @@ call s:set('g:wintabs_ui_sep_inbetween', '|')
 call s:set('g:wintabs_ui_sep_rightmost', '|')
 call s:set('g:wintabs_ui_active_left', ' ')
 call s:set('g:wintabs_ui_active_right', ' ')
+call s:set('g:wintabs_ui_show_tab_name', 0)
```

Comment:
let's call this `g:wintabs_ui_show_vimtab_name` to be consistent with the existing terms, though I don't think I named them well.